rotkehlchen/chain/aggregator.py

```diff
@@ -808,9 +808,7 @@ def _append_evm_account_modification(
             address: ChecksumEvmAddress,  # pylint: disable=unused-argument
     ) -> None:
         """Extra code to run when a non-eth but evm account addition happens"""
-        # If this is the first account added, connect to all relevant nodes
-        chain_manager = self.get_chain_manager(blockchain)
-        chain_manager.node_inquirer.maybe_connect_to_nodes(when_tracked_accounts=False)
+        return
```

Comment:
Then shouldn't this funciton and whenever this function is also used be removed?

We have this code:

```python
        self.chain_modify_append: dict[SupportedBlockchain, Callable[[SupportedBlockchain, BlockchainAddress], None]] = {  # noqa: E501
            SupportedBlockchain.ETHEREUM: self._append_eth_account_modification,  # type:ignore
            SupportedBlockchain.OPTIMISM: self._append_evm_account_modification,  # type:ignore
            SupportedBlockchain.POLYGON_POS: self._append_evm_account_modification,  # type:ignore
            SupportedBlockchain.ARBITRUM_ONE: self._append_evm_account_modification,  # type:ignore
            SupportedBlockchain.BASE: self._append_evm_account_modification,  # type:ignore
            SupportedBlockchain.GNOSIS: self._append_evm_account_modification,  # type:ignore
            SupportedBlockchain.SCROLL: self._append_evm_account_modification,  # type:ignore
        }
```